src/sql/workbench/contrib/connection/browser/connection.contribution.ts

```diff
@@ -128,10 +128,28 @@ configurationRegistry.registerConfiguration({
 			'description': localize('sql.defaultEngineDescription', "Default SQL Engine to use. This drives default language provider in .sql files and the default to use when creating a new connection."),
 			'default': 'MSSQL'
 		},
+		'sql.defaultAuthenticationType': {
+			'type': 'string',
+			'enum': ['SqlAuth', 'AzureMFA', `AzureMFAAndUser`, 'Integrated'],
+			'description': localize('sql.defaultAuthenticationTypeDescription', "Default authentication type to use when connecting to Azure resources. "),
+			'enumDescriptions': [
+				localize('sql.defaultAuthenticationType.SqlAuth', "Sql Login"),
+				localize('sql.defaultAuthenticationType.AzureMFA', "Azure Active Directory - Universal with MFA support"),
+				localize('sql.defaultAuthenticationType.AzureMFAAndUser', ""),
+				localize('sql.defaultAuthenticationType.Integrated', "Windows Authentication"),
+			],
+			'default': 'AzureMFA'
```

Comment:
the default should probably be SqlLogin, since this is still by far more common than the other auth types.

Reply:
Yeap, it's a bit messy.  As you said CI needs to pass, and it needs some cleaning.  Wanted to check it in before it died on the vine.  Happy to see some eyes on it, though!   Let me clean this up ...